### PR TITLE
feat: make history list format configurable

### LINKED ISSUE #1616: Make format for history configurable
I often forget which directory I ran a command from, and when I then find `./start.sh` in my history I'm only halfway there.

With the help of #619 I can find the directory, but I'd like this config:

`history_format` -- default format for `atuin history list`

### PR DESCRIPTION
**Pull request by dongxuwang**
fix #1616 

## Checks
- [x] I am happy for maintainers to push small adjustments to this PR, to speed up the review cycle
- [x] I have checked that there are no existing pull requests for the same thing

### CODE REVIEW

File: atuin-client/config.toml [hunk 1]
```diff
@@ -84,6 +84,9 @@
 ## alt-0 .. alt-9
 # ctrl_n_shortcuts = false
 
+## default history list item format fallback when not appending in cmd
```
Comment (ellie) [inline atuin-client/config.toml:87]:
```suggestion
## default history list format - can also be specified with the --format arg
```

would it also be possible to update this on the docs repo please?

https://github.com/atuinsh/docs

**Review by ellie — COMMENTED** [review]
thanks!

I mentioned in the comment too, but if you could also update the [docs](https://github.com/atuinsh/docs) that would be fantastic 🙏

**Review by ellie — DISMISSED** [review]

**Review by ellie — APPROVED** [review]

**Comment by dongxuwang** [comment]
> thanks!
> 
> I mentioned in the comment too, but if you could also update the [docs](https://github.com/atuinsh/docs) that would be fantastic 🙏

I'll try to update, thanks

**Comment by dlvhdr** [comment]
should this also work in interactive mode? It doesn't seem to and I would love it

**Comment by ben-alkov** [comment]
> should this also work in interactive mode? It doesn't seem to and I would love it

#998

**Pull request merged**